SessionPlayer/HUD.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { View, Text, StyleSheet } from 'react-native';
+
+export default class HUD extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            timer: 0,
+        };
+
+        this.timerIntervalID = setInterval(() => {
+            this.setState({
+                timer: this.state.timer + 1
+            });
+        }, 1000);
+    }
+
+    componentWillUnmount() {
+        clearInterval(this.timerIntervalID);
+    }
+
+    componentDidUpdate() {
+        const { guideState } = this.props;
+
+        if (
+            this.timerIntervalID
+            && (guideState === 'finished' || guideState === 'all_finished')
+        ) {
+            clearInterval(this.timerIntervalID);
+        }
+    }
+
+    render() {
+        const { currentGuideNumber, totalGuides, guideName } = this.props;
+        const { timer } = this.state;
+
+        const minutes = Math.floor(timer / 60);
+        const seconds = timer % 60;
+
+        return (
+            <View style={styles.container}>
+                <Text style={styles.guideName}>
+                    {guideName}
+                </Text>
+                <View style={styles.smallTextContainer}>
+                    <Text style={styles.text}>
```

Comment:
I think rendering both of `current guide` and `timer` should live in their own functions. So `render` could just put them together like:
```
return (
            <View style={styles.container}>
                <Text style={styles.guideName}>
                    {guideName}
                </Text>
                {this.renderGuideNumber()}
                {this.renderGuideTimer()}
            </View>
```
or something like that.